camel-core/src/main/java/org/apache/camel/support/ReloadStrategySupport.java

```diff
@@ -181,6 +181,16 @@ public int getReloadCounter() {
     public int getFailedCounter() {
         return failed;
     }
+    
+    @ManagedOperation(description = "Set succeeded count on xml reload in case overriden")
```

Comment:
this PR is not really needed IMHO as described in the issue. Do you mind if i close it and mark the issue "Won't fix" or do you think we can keep like this and resolve it?